tools/gitleaks/config.toml

```diff
@@ -49,7 +49,7 @@
 	[rules.allowlist]
 		regexes = [
 			'''\"image/png\": \"[0-9a-zA-Z\/+]{1,}(=){0,2}\\n\"''',
-			'''sonarsource/sonarcloud-github-action@'''
+			'''uses: [a-zA-Z0-9_\/-]*@\b[0-9a-f]{40}\b'''
```

Comment:
I don’t think the first character class should include backslashes, and the first `\b` isn’t necessary (the boundary is already fully defined):

```suggestion
			'''uses: [a-zA-Z0-9_/-]*@[0-9a-f]{40}\b'''
```

Reply:
Right, I wondered whether that was the intent — `/` isn’t a special character in regular expressions, and in any case, most special characters don’t need to be escaped in character classes (which is a trick often used to make escapes easier to ready; for example `[.]` instead of `\\.` in scenarios where double escapes are required).